api/anri/settings/djoser.py

```diff
@@ -43,4 +48,38 @@
     },
 }
 
+SIMPLE_JWT = {
```

Comment:
Move to separate module everything related with JWT

Reply:
Done